docs/user_guide.md

```diff
@@ -128,7 +129,21 @@ you can quote only the non-substituted parts:
 "instructions": ${instruction_prefix} "main instruction" ${instruction_suffix}
 ```
 
-You can see a working example here: [../registries/smart_home_include.hocon](../registries/smart_home_include.hocon).
```

Comment:
That example doesn't exist anymore. Replaced by 2 examples, on for json notation and one for hocon notation.